Gems/Atom/RHI/Code/Source/RHI/Image.cpp

```diff
@@ -43,9 +44,9 @@ namespace AZ::RHI
         return static_cast<const ImageFrameAttachment*>(Resource::GetFrameAttachment());
     }
 
-    Ptr<ImageView> Image::BuildImageView(const ImageViewDescriptor& imageViewDescriptor)
+    Ptr<ImageView> Image::GetImageView(const ImageViewDescriptor& imageViewDescriptor)
```

Comment:
This is an api naming change. Any gems calling BuildImageView will not compile and will need to update. We will need to message this api change.

Reply:
Good point, I removed it from this commit (to get the bugfix in earlier), will open a secondary PR that just does the rename and announces the API change properly in advance.